bugbug/bug_features.py

```diff
@@ -205,11 +210,12 @@ def transform(self, bugs):
 
             # TODO: Try simply using all possible fields instead of extracting features manually.
 
-            result = {
-                'data': data,
-                'title': bug['summary'],
-                'comments': ' '.join([c['text'] for c in bug['comments']]),
-            }
+            for cleanup_function in self.cleanup_functions:
+                result = {
```

Comment:
Thanks, the patch is looking better!

A couple of things to fix:
1. The cleanup functions should be applied to both the summary and the comments;
2. The result should not be redefined multiple times, but just once. In the loop you should just apply the cleanup functions to the textual fields.